src/sfizz/Region.h

```diff
@@ -237,6 +245,8 @@ struct Region {
      */
     float getGainToEffectBus(unsigned number) const noexcept;
 
+    const int regionNumber {};
```

Comment:
Since the region still has a default constructor, should we initialize to -1, put an optional, or just disable the default constructor entirely?